aptos-move/vm-genesis/src/lib.rs

```diff
@@ -421,8 +434,9 @@ fn initialize(
     );
 }
 
-fn initialize_features(session: &mut SessionExt) {
-    let features: Vec<u64> = FeatureFlag::default_features()
+fn initialize_features(session: &mut SessionExt, features_override: Option<Vec<FeatureFlag>>) {
```

Comment:
this is not individual override but the whole vec?

Reply:
specifying a patch seems more complicated?

Reply:
yeah just the name is a bit confusing